web/src/common/ui/cards/CardUtilities.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { PropsWithChildren, ReactNode, Fragment } from 'react';
```

Comment:
This is great! Can we add an "isExternal" (boolean) prop and use that instead of trying to programmatically check if the link is outbound?

Reply:
Actually no, that's not a good path forward. The use case that's bugging me is when we pass in something like a map URL, or a user photo URL that ends with .jpg / .png / .bmp

These URLs will be treated as internal router links even tho they should open as external links. I think we'll need to improve "isOutboundLink" implementation to account for that.